functional-test/src/test/groovy/org/rundeck/tests/functional/api/job/JobActivityHistorySpec.groovy

```diff
@@ -0,0 +1,54 @@
+package org.rundeck.tests.functional.api.job
+
+import org.rundeck.util.annotations.APITest
+import org.rundeck.util.container.BaseContainer
+
+@APITest
+class JobActivityHistorySpec extends BaseContainer {
+    def setupSpec() {
+        startEnvironment()
+
+    }
+
+    def "Retrieve job Activity history from Job List Page "() {
+        given: "A Job exists in the project with execution history"
+        def jobId = "3957f735-5269-4c85-bc57-f123598886f6"
```

Comment:
Why would this job exist?

Reply:
> Why would this job exist?

I have used this jobID to represent a job that already exists in the project. This makes it easier to test if the API can get the job's details.  Do I need to create a job first before checking its activity?

Reply:
Your tests should be independent. Given a test suite TS with (optional) setup and tear down functions and multiple tests t1, t2, ..., tn in that suite. You should be able to run `tx` at any time and in any order. 
If you deviate from this rule, there should be a very good reason.

Thus, if your test relies on an job guide, it should be created by the test itself (or setup functionality).